src/lcm/simulation/simulate.py

```diff
@@ -74,7 +80,7 @@ def simulate(
     # value function array is not required, as the agents only consider the current
```

Comment:
Stupid q: Why do we compute/store this in the first place?

Just call this `next_vf_arr`. Also, store as a dict with periods as keys. Not for this PR, ofc.